core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.spec.ts

```diff
@@ -345,13 +348,15 @@ describe('Creator dashboard controller', () => {
         expect(ctrl.relativeChangeInTotalPlays).toBe(5);
       });
 
-      it('should change active tab name', function() {
-        expect(ctrl.activeTab).toBe('myExplorations');
-        ctrl.setActiveTab('suggestions');
-        expect(ctrl.activeTab).toBe('suggestions');
-      });
+      it('should change active tab name when creator clicks on tab',
+        function() {
+          expect(ctrl.activeTab).toBe('myExplorations');
+          ctrl.setActiveTab('suggestions');
+          expect(ctrl.activeTab).toBe('suggestions');
+        });
 
-      it('should set my explorations view correctly', function() {
+      it('should set explorations views format to backend when creator' +
```

Comment:
What would be the pre-check? I'm not seeing it in the other specs so I'm confused.

Reply:
Hi @marianazangrossi -- in this case, your post-check is to `expect(ctrl.myExplorationsView).toBe('a');`

Your pre-check should therefore to be expect that value to be something different, so that you know that its being equal to 'a' is a direct result of the action that was performed in the test.